ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -79,7 +85,26 @@ open class CommandBar: UIView {
 
         super.init(frame: .zero)
 
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(themeDidChange),
+                                               name: .didChangeTheme,
+                                               object: nil)
         configureHierarchy()
+
+        // Update appearance whenever `tokenSet` changes.
+        tokenSetSink = tokenSet.objectWillChange.sink { [weak self] _ in
+            // Values will be updated on the next run loop iteration.
+            DispatchQueue.main.async {
+                self?.updateButtonTokens()
+            }
+        }
+    }
+
+    public override func didMoveToWindow() {
+        super.didMoveToWindow()
+
+        tokenSet.update(fluentTheme)
+        updateButtonTokens()
```

Comment:
Updating the theme will call the sink from line 98, which will also call updateButtonTokens. So we probably should remove it from didMoveToWindow, to avoid calling it more often than we need to.

Reply:
I think we need this for the theme-wide override, removing this doesn't update the tokens when I exit the window.